beacon-chain/core/state/state.go

```diff
@@ -65,11 +65,16 @@ func GenesisBeaconState(deposits []*pb.Deposit, genesisTime uint64, eth1Data *pb
 		}
 	}
 
-	latestBlockRoots := make([][]byte, params.BeaconConfig().HistoricalRootsLimit)
+	latestBlockRoots := make([][]byte, params.BeaconConfig().SlotsPerHistoricalRoot)
```

Comment:
Why change this? It should be HistoricalRootsLimit

Reply:
Are you sure? [Spec says SlotsPerHistoricalRoot.](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#beacon-state)

Reply:
We don't have to follow every naming in the spec, in this example, we found `HistoricalRootsLimit` is more clear than `SlotsPerHistoricalRoot`

Reply:
Ah okay, I'll leave a comment stating that. We should leave comments for any places that we go purposely off-spec.